dbt/sagerx/models/staging/rxclass/stg_rxclass__induces.sql

```diff
@@ -0,0 +1,13 @@
+-- stg_rxclass__induces.sql
+
+with induces as (
+    
+    select
+        *
+    from {{ source('rxclass', 'rxclass') }}
+    where rela = 'induces'
```

Comment:
Similarly to my comment about including the name of the rela_source in the file name, I think it would be more explicit to do like where rela_source = 'X' and rela = 'Y' even if that rela is EXCLUSIVELY from that rela_source. Just makes it more human-understandable, in my opinion.